packages/web/src/components/other/page_not_found.css

```diff
@@ -85,13 +89,14 @@
         display: flex;
         justify-content: center;
         align-items: center;
-        height: 78vh;
+        height: 75vh;
         flex-direction: column-reverse;
     }
     .hubzie-UFO-image {
         width: 100%;
         justify-content: center;
-        margin-bottom: 3%;
+        margin-top: 11%;
+        margin-bottom: 4%;
     }
     .page-not-found-text {
         width: 100%;
```

Comment:
Here add margin-right: 0; On mobile the whole text div is not properly centered because it inherits margin-right: -5% from line 48. Everything else is fine. Again, after making the requested changes don't forget to resolve conversations and re-request review. 

![image](https://user-images.githubusercontent.com/77395265/222103823-c2422310-a885-4339-a67f-cb95d11f60e0.png)